app/components/documents/batch-details/document-details-row.hbs

```diff
@@ -7,11 +7,12 @@
       <AuCheckbox
         @checked={{@isSelected}}
         @onChange={{fn @onSelect @row}}
+        @disabled={{not @row.canBeEditedOrDeleted}}
```

Comment:
This checkbox is disabled, but the top batch editing is still available and will select this disabled checkbox.
![image](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/assets/22245223/49ed5c29-13e5-4fc0-89a1-621597f5296c)

Reply:
I think we need to change the "select all" function to only select rows that do not have the new boolean